README.md

```diff
@@ -161,6 +176,15 @@ Linguist detects the language of a file but the actual syntax-highlighting is po
 If you experience an issue with the syntax-highlighting on GitHub, **please report the issue to the upstream grammar repository, not here.**
 Grammars are updated every time we build the Linguist gem so upstream bug fixes are automatically incorporated as they are fixed.
 
+### I get an error when using Linguist on a directory that is not a Git repo
+
+When specifying a directory Linguist only works on Git repos -- at the root -- and only on committed changes. It's
+primary use is on GitHub.com which uses bare repositories and thus changes need to be committed as individual files don't show on the filesystem.
+
+As a work around you could initialise a temporary Git repo in your directory as demonstrated in this
```

Comment:
We prefer not to abbreviate in documentation outside of code samples.

```suggestion
As a work around you could initialise a temporary Git repository in your directory as demonstrated in this
```

Reply:
I can appreciate that.